.github/workflows/triage.yml

```diff
@@ -221,7 +221,7 @@ jobs:
               keep_if_no_match: true
 
             - label: bump-formula-pr
-              pr_body_content: Created with `brew bump-formula-pr`
+              pr_body_content: Created with `brew bump`
```

Comment:
```suggestion
              pr_body_content: Created with `brew bump
```

will cover all cases

Reply:
oh, right now it does cover all cases, it is just the `bump-formula-pr` can be a false signal when generating autobump list.

Reply:
> it is just the `bump-formula-pr` can be a false signal when generating autobump list.

sorry, I don't understand what this means: can you elaborate a lot more?

Reply:
Sure, right now, folks can use either `bump --open-pr` or `bump-foruma-pr` to create the formula bump PRs, while they both can do the same thing, the `bump-formula-pr` can take lots of parameters overrides, which would be quite manual rather than something can be done by bot (`minio` and `luagit` are examples on this), while AFAIK you can override parameters via `bump --open-pr`, which means, the PRs using it can be also done by bot. 

Let me know if that makes better sense.

Reply:
I understand the differences between the two approaches but not how that affects what should be labelled what or why the contents of the label affects anything?

Reply:
yeah, i have to say, it seems a bit tricky about getting this perfect 😄 

Now, I am gonna manually use [`cannot-autobump` label](https://github.com/Homebrew/homebrew-core/labels/cannot-autobump) to help with the autobump signal.